test/utils/wabt_engine.cpp

```diff
@@ -87,10 +87,7 @@ bool WabtEngine::instantiate(bytes_view wasm_binary)
 
     // Run start function (this will return ok if no start function is present)
     const wabt::interp::ExecResult r = m_executor.RunStartFunction(m_module);
-    if (r.result != wabt::interp::Result::Ok)
-        return false;
-
-    return true;
+    return r.result == wabt::interp::Result::Ok;
```

Comment:
I still hoped to prove both branches are covered with #381, but will add back the inverse of this commit for a test.